### ISSUE TITLE
Create a document for errors

### ISSUE THREAD

**Issue by XVincentX** [stoplightio/prism#527]
Prism's error types are URLs that point nowhere at the moment. Create an error file that lists all the errors and then make sure to create the appropriate redirects in the platform so that people do not a 404 when they're trying to understand what's going on.

**Comment by XVincentX** [comment]
Does the URL format have to be in that way? Can't we set a redirect on the server side somehow that it stays the way it is right now? `https://stoplight.io/prism/errors#no-method-matched`?

The hash is not sent to the server side so we would just need a single redirect.

**Comment by philsturgeon** [comment]
Yeah a [netlify redirect](https://www.netlify.com/docs/redirects/) from `https://stoplight.io/prism/errors` to `https://stoplight.io/p/docs/gh/stoplightio/prism/docs/errors.md` would do the trick.
